diff --git a/drivers/tty/serial/sc16is7xx.c b/drivers/tty/serial/sc16is7xx.c
index 097dace67..ac1d872f1 100644
--- a/drivers/tty/serial/sc16is7xx.c
+++ b/drivers/tty/serial/sc16is7xx.c
@@ -28,7 +28,7 @@
 #include <linux/units.h>
 
 #define SC16IS7XX_NAME			"sc16is7xx"
-#define SC16IS7XX_MAX_DEVS		8
+#define SC16IS7XX_MAX_DEVS		16
 
 /* SC16IS7XX register definitions */
 #define SC16IS7XX_RHR_REG		(0x00) /* RX FIFO */
